### PR TITLE
Set background color

### PR DESCRIPTION
**Pull request by marcguilera**
Just added a convenient way of setting a table bg color.

### CODE REVIEW

**Comment by MobiDevelop** [comment]
I don't think this is something we want to promote doing.

File: gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java [hunk 1]
```diff
@@ -146,6 +151,19 @@ public void setBackground (Drawable background) {
...
 	}
+	
+	/** @param background May be null to clear the background. */
+	public void setBackground(Color color){
+		TextureRegionDrawable d = null;
+		if(color!=null){
+			Pixmap p = new Pixmap(1, 1, Format.RGBA8888);
+			p.setColor(color);
+			p.fill();
+			d = new TextureRegionDrawable(new TextureRegion(new Texture(p)));
```
Comment (MobiDevelop) [inline gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java]: How is this texture going to be disposed?

Comment (marcguilera) [inline gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Table.java]: Yeah that's definitely a problem. I could return it to leave that responsibility to the programmer but I'm not sure if I like this option.

**Comment by nooone** [comment]
I'd say it's probably a pretty rare usecase that you want to have a single color for your table background. And *if* you really want that, it's super easy to do it yourself.

**Comment by xoppa** [comment]
Thanks for submitting your contribution! I'm afraid that we can't pull this in because it creates an (unmanaged) resource which isn't disposed and causes the batch to flush. If you want a solid color for the background then you can use a single pixel texture region, e.g. like [the test skin](https://github.com/libgdx/libgdx/blob/master/tests/gdx-tests-android/assets-raw/skin/white.png) has.

**Pull request closed without merging**